client/components/all-reactos.js

```diff
@@ -0,0 +1,87 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import {connect} from 'react-redux'
+import {Link} from 'react-router-dom'
+import {logout} from '../store'
+import { Icon, Label, Menu, Table } from 'semantic-ui-react'
+
+class AllReactos extends Component {
+
+  handleClick = (evt) => {
+    const cellText = evt.target.innerHTML
+    const reactoId = +cellText.split(' ')[1]
+    this.props.history.push(`/single-reacto/${reactoId}`)
```

Comment:
we can just call with . 'reacto/${reactoId}' instead of 'single-reacto/${reactoId}' . An "id" in a RESTful API usually means a single instance of that type.